core/src/main/java/org/apache/accumulo/core/iterators/user/VisibilityFilter.java

```diff
@@ -100,6 +106,10 @@ public boolean accept(Key k, Value v) {
       try {
         boolean bb = accessEvaluator.canAccess(testVis.toArray());
         cache.put(testVis, bb);
```

Comment:
Could do the following, would need to move the comment up.

```suggestion
        cache.put(new ArrayByteSequence(testVis), bb);
```

Reply:
Are you suggesting doing this instead of changing the testVis reference? Or, in addition to?

Reply:
Was thinking it could be done instead of, thought it made the code a bit shorter.

Reply:
@keith-turner  - I implemented your suggestion in 041bfe7.